server/index.js

```diff
@@ -37,6 +37,43 @@ app.get('/api/photos/workspace/:workspaceId', async (req, res) => {
   }
 });
 
+app.put('/api/photos/workspace/:workspaceId', async (req, res) => {
+  //Test with: curl -X "PUT" -d descriptions="['hipster words','more hipster words']" -d urls="['www.facebook.com','www.example.com']" http://localhost:6001/api/photos/workspace/98
+  const { workspaceId } = req.params;
+  var {descriptions, urls} = req.body;
+
+  descriptions = JSON.parse(descriptions.replace(/'/g, '\"'));
+  urls = JSON.parse(urls.replace(/'/g, '\"'));
+
+  var photos = [];
+  for (var i = 0; i < descriptions.length; i++) {
+    photos.push({
+      id: i,
+      workspaceId: workspaceId,
+      description: descriptions[i],
+      url: urls[i]
```

Comment:
This assumes that descriptions and urls are the same length, or implies that it is okay for either to be undefined.